<launch>
	<arg name="port" default="/dev/ttyUSB0"/>
	<arg name="cube_id" default="1"/>
	<arg name="name" default="my_cube_interface" />
	
	<node pkg="qb_cube_interface" type="qb_cube_interface_node" name="$(arg name)" respawn="true" output="screen">
		
		<!-- REMEMBER TO CHECK THE PORT * -> /dev/ttyUSB*!/-->		
		<!-- AND THEN REMEMBER TO sudo chmod 777 /dev/ttyUSB*!/-->		
		<param name="port" value="$(arg port)"/>

		<!-- 0 for broadcasting, cube_id otherwise/-->
		<param name="cube_id" value="$(arg cube_id)"/> 
	</node>
</launch>

	
